qancode/comparisons.py

```diff
@@ -264,7 +283,8 @@ def compare_data(self):
             result = self.compute_image_difference()
             print('Distance metric: {}'.format(self.diff_distance_metric))
             return result
-        except IndexError:
-            print('{}COMPARISON ERROR. SKIPPING.{}'.format(bcolors.FAIL, bcolors.ENDC))
+        except:
```

Comment:
```
except Exception:
```
Not having the exception is a pep8 violation.

Reply:
The operation before this is so expensive that I really would like a catch-all here. No matter what the exception I would like for it to try to continue. Maybe there is a middle way that avoids importing obscure OpenCV exceptions? On the other hand maybe we can just go back to using IndexError since I fixed what was causing the OpenCV exception.